code/modules/psionics/faculties/coercion.dm

```diff
@@ -237,7 +255,6 @@
 		return FALSE
 	. = ..()
 	if(.)
-		user.visible_message(SPAN_NOTICE("<i>[user] touches their fingers to their temple.</i>"))
```

Comment:
No, this is staying in. I am **absolutely never** approving invisible telepathy, full stop.

Reply:
Readded it with a different, less 1950s B-Movie message.